### DISCUSSION TITLE
Ability to compare selected history versions

### DISCUSSION THREAD

**Discussion by chausmann-weptun** [outline/outline#7440] [Ideas]
Currently, you can open a page's history view and select a single entry from the list. Outline will display that version and highlight the changes made. However, it is not possible to generate a summary of all changes across multiple updates. For example, you cannot select version 1 and version 4 and then get a summary of all changes made in versions 2, 3, and 4.

**Comment by tommoor** [comment]
It's a good idea 👍 – pretty easy to achieve on the backend, just a matter of UI to pick which version you're comparing against.
